accounts/models.py

```diff
@@ -11,10 +11,11 @@ class User(AbstractUser):
     created_at = models.DateTimeField(auto_now_add=True)
 
     def __str__(self):
```

Comment:
Add doc string here @Yshah112  @rinkishivani

Reply:
added docstring in feature/coomon